src/Locale/en/Bed.json

```
@@ -0,0 +1,6 @@
+{
+    "bed_search_placeholder": "Search by beds name",
+    "BED_WITH_OXYGEN_SUPPORT": "BED WITH OXYGEN SUPPORT",
+    "REGULAR": "REGULAR",
```

Need not be uppercased.

```suggestion
    "BED_WITH_OXYGEN_SUPPORT": "Bed with Oxygen Support",
    "REGULAR": "Regular",
```